interfaces/builtin/account_control_test.go

```diff
@@ -35,9 +36,23 @@ type AccountControlSuite struct {
 	plug  *interfaces.Plug
 }
 
-var _ = Suite(&AccountControlSuite{
-	iface: builtin.NewAccountControlInterface(),
-	slot: &interfaces.Slot{
+var _ = Suite(&AccountControlSuite{})
+
+const accountCtlMockPlugSnapInfo = `name: other
+version: 1.0
+plugs:
```

Comment:
Ah I see.. I'd keep it as is, I'm sure we have tests for this being optional somewhere else. No harm in being explicit here.